lib/block-supports/duotone.php

```diff
@@ -311,6 +311,9 @@ function gutenberg_get_duotone_filter_id( $preset ) {
  * @return string        Duotone CSS filter property url value.
  */
 function gutenberg_get_duotone_filter_property( $preset ) {
+	if ( isset( $preset['colors'] ) && 'unset' === $preset['colors'] ) {
+		return 'none';
+	}
```

Comment:
`gutenberg_get_duotone_filter_property` is used externally by `WP_Theme_JSON`, so we want to keep the original behavior.

Reply:
Oh man. Its so horrible having to remember that everything is public in PHP. Thank you for catching this.